src/Affix.ts

```diff
@@ -0,0 +1,15 @@
+import {Stat} from './Stat'
+
+interface IAffix {
+  name: string
+  affixType: string
+  stats: Stat
+}
+
+export class Affix implements IAffix {
+  constructor(
+    public name: string,
```

Comment:
Make sure to get this compiling locally so that you can iterate to fix these build failures